packages/immutadot/src/nav/propNav.js

```diff
@@ -21,6 +21,17 @@ class PropNav extends ObjectNav {
     copy[this.key] = this.next.update(updater)
     return copy
   }
+
+  unset() {
+    const copy = this.copy()
+
+    if (this.next.final)
```

Comment:
This is barely the same code as in the indexNav. Can we do something about it ?

Reply:
Yes this is exactly the same code as in IndexNav, and we could mutualize it in their common ancestor BaseNav.
But for comprehension sake, I think it is better to have two copies of this method, one that uses an `index` which denotes an array access, and one that uses a `key` which denotes a property access.